example/src/shared/utils.ts

```diff
@@ -25,3 +25,26 @@ export const checkIfUrl = (val: string) => {
 export const extractRoomNameFromUrl = (url: string) => {
   return url.substring(url.indexOf(VIDEOROOM_URL) + VIDEOROOM_URL.length);
 };
+
+export const checkIfArraysAreTheSame = (
```

Comment:
https://lodash.com/docs/4.17.15#isEqual ?

Reply:
I was looking at docs and couldn't find this function..., will use it now. thanks